egs/aidatatang_200zh/asr1/conf/tuning/decode_pytorch_transformer.yaml

```diff
@@ -0,0 +1,8 @@
+batchsize: 0
+beam-size: 10
+penalty: 0.0
+maxlenratio: 0.0
+minlenratio: 0.0
+ctc-weight: 0.5
+lm-weight: 0.7
```

Comment:
I feel that `lm-weight` is a bit too large.
Maybe you can tune `ctc-weight` and `lm-weight` a bit with the dev set (and probably `ngram-weight` as well)

Reply:
Should I try tuning this in a later PR?

Reply:
yes, let's do it later.
let's try to finish this PR anyway.

Reply:
I've added the description to https://github.com/espnet/espnet/blob/master/egs/README.md